daiquiri/formatter.py

```diff
@@ -127,30 +138,28 @@ def __new__(cls, name, bases, dct):
 
 class Swatches(six.with_metaclass(SwatchMeta, object)):
     """ Collection of default colors. """
-    Default = Palette(32, 36, bold(33), bold(31), bold(31), base=True)
 
-    Bright = Palette(226, 220, 214, 202, bold(196))
-    Blue = Palette(39, 32, 25, 24, bold(23))
-    Cool = Palette(33, 45, 172, 203, bold(125))
-    Purple = Palette(183, 141, 171, 90, bold(57))
-    Warm = Palette(184, 154, 202, 208, bold(160))
+    # colors from the user defined tty color palette
+    Default = Palette(32, 36, Palette.bold(33), Palette.bold(31), Palette.bold(31), base=True)
+
+    # colors from the 256-color table
+    Blue = Palette(39, 32, 25, 24, Palette.bold(23))
+    Bright = Palette(226, 220, 214, 202, Palette.bold(196))
+    Cool = Palette(33, 45, 172, 203, Palette.bold(125))
+    Purple = Palette(183, 141, 171, 90, Palette.bold(57))
+    Warm = Palette(184, 154, 202, 208, Palette.bold(160))
 
 
 class ColorFormatter(logging.Formatter):
-    LEVEL_COLORS = None
     COLOR_STOP = '\033[0m'
 
-    def __init__(self, fmt=None, datefmt=None, style='%', palette=Swatches.Default):
-        super(ColorFormatter, self).__init__(fmt, datefmt, style)
+    def __init__(self, fmt=None, datefmt=None, palette=Swatches.Default):
+        super(ColorFormatter, self).__init__(fmt, datefmt)
```

Comment:
`style='%'` is Python3 only it seems. It's used to define what type of string formatting the logging module should use.